internal/configuration/validator/configuration_test.go

```diff
@@ -99,3 +99,33 @@ func TestShouldAddDefaultAccessControl(t *testing.T) {
 	assert.NotNil(t, config.AccessControl)
 	assert.Equal(t, "deny", config.AccessControl.DefaultPolicy)
 }
+
+func TestShouldRaiseErrorWhenSSLCertWithoutKeyIsProvided(t *testing.T) {
+	validator := schema.NewStructValidator()
+	config := newDefaultConfig()
+	config.SSLCert = "/tmp/cert.pem"
+
+	Validate(&config, validator)
+	require.Len(t, validator.Errors(), 1)
+	assert.EqualError(t, validator.Errors()[0], "SSL certificate provided but no key")
```

Comment:
Update per above recommendations:

```suggestion
	assert.EqualError(t, validator.Errors()[0], "No SSL key provided, please check the "ssl_key" which has been configured")
```